fiftyone/core/dataset.py

```
@@ -9388,12 +9404,18 @@ def _create_dataset(
     persistent=False,
+    media_type=None,
     _patches=False,
     _frames=False,
     _clips=False,
     _src_collection=None,
 ):
     slug = _validate_dataset_name(name)
+    if media_type and media_type not in fom.MEDIA_TYPES:
+        raise ValueError(
+            "Invalid media type '%s'. Must be one of %s"
+            % (media_type, fom.MEDIA_TYPES)
```

Ruff (0.15.15)

[warning] Use format specifiers instead of percent format

Replace with format specifiers

(UP031)